packages-nextgen/kestrel_core/src/kestrel/frontend/mapping/entity/stix.yaml

```diff
@@ -0,0 +1,24 @@
+# TODO mapping for artifact:
+autonomous-system: organization
+directory: file
+domain-name: endpoint
+email-addr: user
+email-message: email
+file: file
+ipv4-addr: endpoint
+ipv6-addr: endpoint
+mac-addr: endpoint
+network-traffic: network-activity
+process: process
+software: product
+url: http_request
+user-account: user
+x-ibm-finding: security-finding
+x-ibm-ttp-tagging: attack
+x-oca-asset: device
+# TODO: mapping for x-oca-event
+x-oca-event:
```

Comment:
How about mapping it to [Base Event](https://schema.ocsf.io/1.0.0/base_event?extensions=)?

I am thinking to add two alias to it in Kestrel: `event` and `activity`.

The only issue is this is good for query translation, not data translation (another example below on `file.hashes`).

Reply:
Mapped `x-oca-event` to `event` and `activity` (virtual) classes in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/414/commits/15c9caec06163a26299565dfdf151fcf4509fcfa